docs/api.md

```diff
@@ -725,6 +735,40 @@ Determines whether or not boss bar creates fog
 
 Determines what color the boss bar color is, one of `pink`, `blue`, `red`, `green`, `yellow`, `purple`, `white`
 
+### mineflayer.Particle
+
+#### Particle.id
+
+Particle ID, as defined in the [protocol](https://wiki.vg/Protocol#Particle)
+
+#### Particle.name
+
+Particle Name, as defined in the [protocol](https://wiki.vg/Protocol#Particle)
+
+#### Particle.longDistance
+
+Determines whether or not to increase the particle distance
+
+#### Particle.position
+
+Vec3 instance of where the particle was created
+
+#### Particle.offset
+
+Vec3 instance of the particle's offset
+
+#### Particle.particleData
```

Comment:
This type of data should be abstracted away, not exposed directly. It's likely to break between versions and between different editions of Minecraft (eg bedrock). Try to only expose data/fields that would make sense to an ingame player.